packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -369,6 +369,9 @@ export default class InteractSubmode extends Component {
       allRealmsInfo: async () => {
         return await here.realm.allRealmsInfo;
       },
+      fetchCard: async (url: string) => {
+        return await here.store.get(url);
```

Comment:
is the skill card in the identity context already? since its in base...

In which case, we prob want to use `store.peek` since that just looks into the identity map. There is no issue in using that bcos the `remixSkillCard` is consumed straight away